Tests/SentryTests/SwiftDescriptorTests.swift

```diff
@@ -28,7 +28,61 @@ class SwiftDescriptorTests: XCTestCase {
         XCTAssertEqual(name, "InnerClass")
     }
     
+    func testgetSwiftErrorDescription_EnumValue() {
+        let actual = SwiftDescriptor.getSwiftErrorDescription(SentryTestError.someError)
+        XCTAssertEqual("someError", actual)
+    }
+    
+    func testgetSwiftErrorDescription_EnumValueWithData() {
+        let actual = SwiftDescriptor.getSwiftErrorDescription(SentryTestError.someOhterError(10))
+        XCTAssertEqual("someOhterError", actual)
+    }
+    
+    func testgetSwiftErrorDescription_StructWithData() {
+        let actual = SwiftDescriptor.getSwiftErrorDescription(XMLParsingError(line: 10, column: 12, kind: .internalError))
+        XCTAssertNil(actual)
+        
+        SentrySDK.capture(error: LoginError.wrongPassword)
```

Comment:
m: What is the purpose of this line in this test?

Reply:
It was there by accident. Good catch thanks.